src/main/kotlin/com/manimdsl/errorhandling/ErrorHandler.kt

```diff
@@ -18,6 +18,15 @@ object ErrorHandler {
         semanticErrors.add("Semantic error at $linePos: $errorEvent")
     }
 
+    fun addRuntimeError(errorEvent: String, lineNumber: Int) {
+        println(
+                "Error detected during program execution. Animation could not be generated. \n" +
+                        "Exit code: ${ExitStatus.RUNTIME_ERROR.code}"
```

Comment:
Since we do `exitProcess` with this error code in `Compiler.kt` I don't think we need to print the error code here right?

Reply:
The exitProcess just returns the error code to the shell it's the same pattern as other errors

Reply:
Also it's not printed for any of the other types of error